docker-compose.yml

```diff
@@ -59,6 +59,7 @@ services:
       - TPP_URL=http://open-banking-gateway:8085
       - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:5432/open_banking?currentSchema=fintech
 
+      - FINTECH-UI_HOST=http://localhost:24200
```

Comment:
After you have introduced FINTECH-UI_HOST 
FINTECH-UI_REDIRECT-URL
FINTECH-UI_UNAUTHORIZED-URL
FINTECH-UI_EXCEPTION-URL
can be removed